packages/component/src/Attachment/Text/private/shortenURL.spec.ts

```diff
@@ -0,0 +1,20 @@
+import shortenURL from './shortenURL';
+
+test.each([
+  ['https://www.example.com/', 'example.com'],
+  ['https://www.example.com/index.html', 'example.com'],
+  ['HTTPS://WWW.EXAMPLE.COM/index.html', 'example.com'],
+  ['https://www.example.co.jp/index.html', 'example.co.jp'],
+  ['https://www.example.com:443/index.html', 'example.com'],
+  ['http://www.example.com:80/index.html', 'example.com'],
+  ['https://www.support.example.com/index.html', 'support.example.com'],
+  ['https://www.microsoft/index.html', 'microsoft'],
+  ['https://www.例子.com/index.html', 'xn--fsqu00a.com'],
+  ['https://www.example.com:8443/index.html', 'www.example.com:8443'],
```

Comment:
If the url comes in as https://www.example.com:8080 would we want to strip the protocol?

Reply:
rules say:


https://www.example.com:8443/index.html | www.example.com:8443 | No, not default port number

Suggesting default ports would be stripped?

Reply:
No, `https://www.example.com:8080/` will become `www.example.com:8080`, not stripping the port.

Reply:
Added a test to show the port number 8080 is not stripped.